pom.xml

```diff
@@ -30,5 +30,12 @@
             <version>3.141.59</version>
         </dependency>
 
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>4.2.2</version>
```

Comment:
Please, create separate `<properties> </properties>` section in your pom.xml and move all dependencies versions to this section.

Reply:
Created